.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: node:8
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run: yarn install
+
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+        
+      # run tests!
+      # - run: yarn lint
+      - run: yarn test
+
+      - run:
+          name: Building site
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
Ram has a slightly old version of this. Branches and configuration should be configured through variables. see this https://github.com/developmentseed/project-seed/blob/master/.circleci/config.yml